reactor-core/src/main/java/reactor/core/publisher/MonoSingleOptional.java

```diff
@@ -25,9 +25,8 @@
 import reactor.util.context.Context;
 
 /**
- * Expects and emits a single item from the source wrapped into an Optional, emits
- * an empty Optional instead for empty source or signals
- * IndexOutOfBoundsException for a multi-item source.
+ * Emits a single item from the source wrapped into an Optional, emits
```

Comment:
I see you improved the javadoc. I think it looks good, please also apply this same principle to the `Callable` operator.